docker-compose.yml

```diff
@@ -71,10 +77,11 @@ services:
           context: .
           dockerfile: Dockerfile.crossdock
         ports:
-            - "8080-8088"
+            - "8080-8090"
         environment:
             - REDIS=enabled
             - CHERAMI=enabled
+            - GRPC=enabled
```

Comment:
I was just following the pattern, I will remove, also note https://github.com/yarpc/yarpc-go/issues/916